packages/frontend/core/src/modules/collection-rules/impls/group-by/date.ts

```
@@ -0,0 +1,35 @@
+import type { GroupByParams } from '../../types';
+
+export class DatePropertyGroupByProvider
+  extends Service
+  implements GroupByProvider
+{
+  constructor(private readonly docsService: DocsService) {
+    super();
+  }
+
+  groupBy$(
+    _items$: Observable<Set<string>>,
```

ESLint

[error] Finnish notation should not be used here.

(rxjs/finnish)